yang/ietf-acl-enh.yang

```diff
@@ -91,6 +91,96 @@ module ietf-acl-enh {
        header.";
   }
 
+  identity tcp-flag {
+    description
+      "Base Identity for the TCP Flags.";
+  }
+
+  identity ACK {
+    base tcp-flag;
+    description
+      "The acknowledgment ACK TCP flag is used to acknowledge the 
+      successful receipt of a packet. As we can see from the diagram above, 
+      the receiver sends an ACK as well as a SYN in the second step
+      of the three-way handshake process to tell the sender 
+      that it received its initial packet.";
+  }
+
+  identity SYN {
+    base tcp-flag;
+    description
+      "The Synchronize sequence numbers flag is used to establish a three-way 
+      handshake between two hosts. Only the first packet from both the sender 
+      and receiver should have this flag set. The following diagram 
+      illustrates a three-way handshake process.";

```

Comment:
Please replace the description with the text from 3.1 of 9293 provided in the previous comment. Thanks.